draft-ietf-add-split-horizon-authority.xml

```diff
@@ -350,6 +361,28 @@
     </section>
     <!-- Validating -->
 
+    <section>
+      <name>Delegating DNSSEC across Split DNS Boundaries</name>
+      <t>We wish to enable DNSSEC validation of local DNS names without requiring
+      the local resolver to hold DNSSEC private keys that are valid for the parent
+      zone.  To support this configuration, parent zones <bcp14>MAY</bcp14> add a
+      "ds=..." key to the Verification Record whose value is the RDATA of a single
+      DS record, base64url-encoded. This DS record authorizes a DNSKEY whose Owner
+      Name is "resolver.arpa."</t>
+      <t>To validate DNSSEC, the client first fetches and validates the Verification
```

Comment:
I don't get how the proposed DNSSEC validation in this section would prevent an malicious network from spoofing the TXT record in DNS responses. An on-path attacker acting as resolver could spoof the key in "ds" to claim ownership of a domain. How will the attack be detected by the client ?

Reply:
Spoofing of the TXT record during Tamperproof Resolution would be prevented by ordinary DNSSEC RRSIGs, like any ordinary TXT record.

Reply:
I am not able to understand the use the DS key. It would be helpful to illustrate with an example.

Reply:
OK, I added an example.

Reply:
Thanks. You may want to delete the following lines:
1. Today, on the Internet it publishes two NS records, "ns1.example.com" and "ns2.example.com".
2. This specification relies on securely identified local DNS servers and globally valid NS records.

Reply:
OK, done